docs/search/criteria_reference/isvirtual_criterion.md

```diff
@@ -0,0 +1,41 @@
+# IsVirtual Criterion
+
+The `IsVirtual` Search Criterion searches for virtual or physical products. 
+
+## Arguments
+
+- (optional) `value` - bool representing whether to search for virtual (default `true`)
```

Comment:
There is no "value" argument: https://github.com/ibexa/product-catalog/pull/1025/files#diff-05ba6990cb58e046d2e1b37f039baf002c696413d8841c7e1411c0f9b8b1bcc8R17